lib/commands/app-management.js

```diff
@@ -55,34 +55,14 @@ commands.queryAppState = async function queryAppState (appId) {
  */
 commands.activateApp = async function activateApp (appId) {
   this.log.debug(`Activating '${appId}'`);
-  const apiLevel = await this.adb.getApiLevel();
-  // Fallback to Monkey in older APIs
-  if (apiLevel < 24) {
-    // The monkey command could raise an issue as https://stackoverflow.com/questions/44860475/how-to-use-the-monkey-command-with-an-android-system-that-doesnt-have-physical
-    // but '--pct-syskeys 0' could cause another background process issue. https://github.com/appium/appium/issues/16941#issuecomment-1129837285
-    const cmd = ['monkey',
-      '-p', appId,
-      '-c', 'android.intent.category.LAUNCHER',
-      '1'];
-    let output = '';
-    try {
-      output = await this.adb.shell(cmd);
-      this.log.debug(`Command stdout: ${output}`);
-    } catch (e) {
-      this.log.errorAndThrow(`Cannot activate '${appId}'. Original error: ${e.message}`);
-    }
-    if (output.includes('monkey aborted')) {
-      this.log.errorAndThrow(`Cannot activate '${appId}'. Are you sure it is installed?`);
-    }
-    return;
-  }
```

Comment:
Android 6:

with latest adb (without monkey):
```
[HTTP] --> POST /wd/hub/session/ee2ffcf9-bae6-411c-bc97-5c414e05faad/appium/device/activate_app
[HTTP] {"bundleId":"io.appium.android.apis"}
[debug] [AndroidUiautomator2Driver@5aad (ee2ffcf9)] Calling AppiumDriver.activateApp() with args: ["io.appium.android.apis",null,null,null,"ee2ffcf9-bae6-411c-bc97-5c414e05faad"]
[debug] [AndroidUiautomator2Driver@5aad (ee2ffcf9)] Activating 'io.appium.android.apis'
[debug] [ADB] Running '/Users/kazuaki/Library/Android/sdk/platform-tools/adb -P 5037 -s CB5A22LZ79 shell dumpsys package io.appium.android.apis'
[debug] [ADB] Running '/Users/kazuaki/Library/Android/sdk/platform-tools/adb -P 5037 -s CB5A22LZ79 shell am start -a android.intent.action.MAIN -c android.intent.category.LAUNCHER -f 0x10200000 -n io.appium.android.apis/.view.Buttons1'
[debug] [AndroidUiautomator2Driver@5aad (ee2ffcf9)] Starting: Intent { act=android.intent.action.MAIN cat=[android.intent.category.LAUNCHER] flg=0x10200000 cmp=io.appium.android.apis/.view.Buttons1 }
[debug] [AndroidUiautomator2Driver@5aad (ee2ffcf9)] Warning: Activity not started, its current task has been brought to the front
[debug] [AndroidUiautomator2Driver@5aad (ee2ffcf9)] Responding to client with driver.activateApp() result: null
[HTTP] <-- POST /wd/hub/session/ee2ffcf9-bae6-411c-bc97-5c414e05faad/appium/device/activate_app 200 1298 ms - 14
```

logcat:
```
05-22 22:09:52.538   953  3528 I ActivityManager: START u0 {act=android.intent.action.MAIN cat=[android.intent.category.LAUNCHER] flg=0x10200000 cmp=io.appium.android.apis/.view.Buttons1} from uid 2000 on display 0
```


The monkey way:
```
05-22 22:15:59.131   953  3528 I ActivityManager: START u0 {act=android.intent.action.MAIN cat=[android.intent.category.LAUNCHER] flg=0x10200000 cmp=io.appium.android.apis/.ApiDemos} from uid 2000 on display 0
```

Hm, probably it is safe to keep the monkey way in regression perspective